src/Objs/Redirect/RedirectComponent.tsx

```diff
@@ -30,11 +30,19 @@ provideComponent(Redirect, ({ page }) => {
     })
   }, [requireLogin, link])
 
-  if (!link) {
+  if (!link && isEditorLoggedIn()) {
     return (
-      <InPlaceEditingPlaceholder center>
-        Select a link in the redirect page properties.
-      </InPlaceEditingPlaceholder>
+      <div className="container d-flex h-100">
+        <div className="alert alert-warning d-flex m-auto">
+          <i className="bi bi-exclamation-circle bi-2x" aria-hidden="true"></i>
+          <div className="my-auto mx-2">
+            This obj never renders any content but triggers a redirect to the
+            specified target.
```

Comment:
* [ ] FYI Depending on which behavior we agree on, instead of "specified target" we could render a link to the target. This way, the editor can simulate the redirect by clicking.

Reply:
Please do the fine-tuning later. This PR is just the basic implementation.